app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt

```diff
@@ -321,6 +322,15 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasSupportF
                     .load(account.header)
                     .into(accountHeaderImageView)
 
+            accountAvatarImageView.setOnClickListener {
+                val intent = ViewMediaActivity.newAvatarIntent(it.context, account.avatar)
+
+                ViewCompat.setTransitionName(it, account.avatar)
```

Comment:
IMHO this lambda is too big to use `it`, it takes some effort to understand what it means